### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by ariya**
See also #1527 for details.

### CODE REVIEW

**Comment by hzoo** [comment]
Exciting 🎉 !

File: README.md [hunk 1]
```diff
@@ -12,17 +12,37 @@ with the help of [many contributors](https://github.com/jquery/esprima/contribut
 
 ### Features
 
-- Full support for ECMAScript 6 ([ECMA-262](http://www.ecma-international.org/publications/standards/Ecma-262.htm))
+- Full support for ECMAScript 2016 ([ECMA-262 7th Edition](http://www.ecma-international.org/publications/standards/Ecma-262.htm))
 - Sensible [syntax tree format](https://github.com/estree/estree/blob/master/es5.md) as standardized by [ESTree project](https://github.com/estree/estree)
 - Experimental support for [JSX](https://facebook.github.io/jsx/), a syntax extension for [React](https://facebook.github.io/react/)
```
Comment (hzoo) [inline README.md]: Is this still going to be marked as "experimental"?

File: package.json [hunk 2]
```diff
@@ -92,6 +92,7 @@
     "dynamic-analysis": "npm run analyze-coverage && npm run check-coverage",
     "compile": "tsc -p src/ && webpack && node tools/fixupbundle.js",
     "test": "npm run compile && npm run all-tests && npm run static-analysis && npm run dynamic-analysis",
+    "prepublish": "npm run compile",
```
Comment (ariya) [inline package.json]: See if I got this right @mikesherov.

* `dist/esprima.js` in the `files` field of package.json
* `dist` is **not** listed in `.npmignore`

File: README.md (hunk 1 above)
Comment (ariya) [inline README.md]: I think it's a fair warning, consider that feature has not been widely used yet.

File: package.json (hunk 2 above)
Comment (mikesherov) [inline package.json]: I think that's correct. We can always to a beta release to ensure it all went right!

Comment (ariya) [inline package.json]: Thanks @mikesherov!

About the beta release: "I  don't always test my code but when I do, I do it in production".  😸

Comment (mikesherov) [inline package.json]: LOLOL

**Comment by mikesherov** [comment]
👍

**Pull request closed without merging**